pom.xml

```diff
@@ -364,6 +364,14 @@
         <type>pom</type>
         <scope>import</scope>
       </dependency>
+      <dependency>
```

Comment:
netty-bom  has been seted in parent pom.xml 
and using in linkis-commons/linkis-module/pom.xml with 
```
    <dependency>
      <groupId>io.netty</groupId>
      <artifactId>netty-all</artifactId>
    </dependency>
```
There is no need to add this configuration